### DISCUSSION TITLE
Unable to connect to postal server using Outlook or telnet

### DISCUSSION THREAD

**Discussion by ttechnomarkets** [postalserver/postal#1585] [Help with using Postal]
I'm trying to connect with SMTP and I'm not able to use the provided details in the configuration. Please advise, how have you been able to successfully send via SMTP? What's the process?

1. If i do telnet in local host it's working.
2. if i send email using postal web interface, Email's working.
3. if i do telnet from remote system: not connecting.

Last login: Fri Sep  3 07:19:42 2021 from 157.45.200.105
root@ns100237:~# netstat -lnp
Active Internet connections (only servers)
Proto Recv-Q Send-Q Local Address           Foreign Address         State       PID/Program name
tcp        0      0 127.0.0.53:53           0.0.0.0:*               LISTEN      2987/systemd-resolv
tcp        0      0 127.0.0.1:53            0.0.0.0:*               LISTEN      1351/named
tcp        0      0 0.0.0.0:22              0.0.0.0:*               LISTEN      20401/sshd
tcp        0      0 127.0.0.1:953           0.0.0.0:*               LISTEN      1351/named
tcp        0      0 127.0.0.1:2019          0.0.0.0:*               LISTEN      18280/caddy
tcp        0      0 127.0.0.1:5000          0.0.0.0:*               LISTEN      5224/[postal]
tcp        0      0 127.0.0.1:5672          0.0.0.0:*               LISTEN      16311/docker-proxy
tcp        0      0 127.0.0.1:3306          0.0.0.0:*               LISTEN      15976/docker-proxy
tcp6       0      0 :::80                   :::*                    LISTEN      18280/caddy
tcp6       0      0 ::1:53                  :::*                    LISTEN      1351/named
tcp6       0      0 :::22                   :::*                    LISTEN      20401/sshd
tcp6       0      0 :::25                   :::*                    LISTEN      5390/rake postal:sm
tcp6       0      0 ::1:953                 :::*                    LISTEN      1351/named
tcp6       0      0 :::443                  :::*                    LISTEN      18280/caddy
udp        0      0 127.0.0.53:53           0.0.0.0:*                           2987/systemd-resolv
udp        0      0 127.0.0.1:53            0.0.0.0:*                           1351/named
udp6       0      0 ::1:53                  :::*                                1351/named
Active UNIX domain sockets (only servers)
Proto RefCnt Flags       Type       State         I-Node   PID/Program name     Path
unix  2      [ ACC ]     SEQPACKET  LISTENING     16626    1/systemd            /run/udev/control
unix  2      [ ACC ]     STREAM     LISTENING     20024343 2698/systemd         /run/user/0/systemd/private
unix  2      [ ACC ]     STREAM     LISTENING     45882    15993/containerd-sh  /run/containerd/s/e2d1f3a06a89c15d7b08d16910bd822ea0b67e93a89c6a61a3e6ae49e82de28d
unix  2      [ ACC ]     STREAM     LISTENING     20024347 2698/systemd         /run/user/0/snapd-session-agent.socket
unix  2      [ ACC ]     STREAM     LISTENING     20024348 2698/systemd         /run/user/0/gnupg/S.gpg-agent.browser
unix  2      [ ACC ]     STREAM     LISTENING     20024349 2698/systemd         /run/user/0/gnupg/S.gpg-agent.extra
unix  2      [ ACC ]     STREAM     LISTENING     20024350 2698/systemd         /run/user/0/gnupg/S.gpg-agent
unix  2      [ ACC ]     STREAM     LISTENING     20024351 2698/systemd         /run/user/0/gnupg/S.dirmngr
unix  2      [ ACC ]     STREAM     LISTENING     20024352 2698/systemd         /run/user/0/gnupg/S.gpg-agent.ssh
unix  2      [ ACC ]     STREAM     LISTENING     52534    16329/containerd-sh  /run/containerd/s/7ea5e77b479c1e36f8a4a3bbe419fb192d2b5c3297be4de4b27f52344d97f626
unix  2      [ ACC ]     STREAM     LISTENING     20449    1199/irqbalance      @irqbalance1199.sock
unix  2      [ ACC ]     STREAM     LISTENING     20039445 5168/containerd-shi  /run/containerd/s/f06f3bbbe2416ec3a71c0051d2e8d4a8d31272be2475cf90c694753315a42b39
unix  2      [ ACC ]     STREAM     LISTENING     20042801 5233/containerd-shi  /run/containerd/s/a2884d4a064a0d4293289e9be438f5258718eb8b12d0ec3ca5ed6dc6c78bb10b
unix  2      [ ACC ]     STREAM     LISTENING     20041527 5285/containerd-shi  /run/containerd/s/beb84b61ce46ad2b8a4c14cd853a9e84619a33a1846463513bad4640efdfef3b
unix  2      [ ACC ]     STREAM     LISTENING     23312    1/systemd            /run/systemd/private
unix  2      [ ACC ]     STREAM     LISTENING     20036131 5336/containerd-shi  /run/containerd/s/b1097167e2aa9bd78fba5bcbfd7ddd59b71258a0b2655977aaf7bdc2811704f4
unix  2      [ ACC ]     STREAM     LISTENING     43801    1/systemd            /var/run/docker.sock
unix  2      [ ACC ]     STREAM     LISTENING     20044813 5414/containerd-shi  /run/containerd/s/7ff66806abeb4bbb949d4c8fcac662c5ed51616a8e1d563471ae7dc13dbb101b
unix  2      [ ACC ]     STREAM     LISTENING     16606    1/systemd            /run/systemd/fsck.progress
unix  2      [ ACC ]     STREAM     LISTENING     16617    1/systemd            /run/lvm/lvmetad.socket
unix  2      [ ACC ]     STREAM     LISTENING     16619    1/systemd            /run/systemd/journal/stdout
unix  2      [ ACC ]     STREAM     LISTENING     76651    18252/containerd-sh  /run/containerd/s/6488151750ef7c962e28edc6f5ab9ece1ff97fd52166763758b9e9fb997dcdb9
unix  2      [ ACC ]     STREAM     LISTENING     16635    1/systemd            /run/lvm/lvmpolld.socket
unix  2      [ ACC ]     STREAM     LISTENING     21940    1/systemd            /var/run/dbus/system_bus_socket
unix  2      [ ACC ]     STREAM     LISTENING     21953    1/systemd            /run/uuidd/request
unix  2      [ ACC ]     STREAM     LISTENING     21955    1/systemd            /run/acpid.socket
unix  2      [ ACC ]     STREAM     LISTENING     21957    1/systemd            /run/snapd.socket
unix  2      [ ACC ]     STREAM     LISTENING     21959    1/systemd            /run/snapd-snap.socket
unix  2      [ ACC ]     STREAM     LISTENING     20365    1/systemd            @ISCSIADM_ABSTRACT_NAMESPACE
unix  2      [ ACC ]     STREAM     LISTENING     47355    14370/containerd     /run/containerd/containerd.sock.ttrpc
unix  2      [ ACC ]     STREAM     LISTENING     47356    14370/containerd     /run/containerd/containerd.sock
unix  2      [ ACC ]     STREAM     LISTENING     35800    14518/dockerd        /var/run/docker/metrics.sock
unix  2      [ ACC ]     STREAM     LISTENING     21944    1/systemd            /var/lib/lxd/unix.socket
unix  2      [ ACC ]     STREAM     LISTENING     42780    14518/dockerd        /var/run/docker/libnetwork/e9a99888affe.sock
root@ns100237:~#

root@ns100237:~# postal status
      Name                     Command               State   Ports
------------------------------------------------------------------
postal_cron_1       /docker-entrypoint.sh post ...   Up
postal_requeuer_1   /docker-entrypoint.sh post ...   Up
postal_smtp_1       /docker-entrypoint.sh post ...   Up
postal_web_1        /docker-entrypoint.sh post ...   Up
postal_worker_1     /docker-entrypoint.sh post ...   Up
root@ns100237:~#

Tried disabling postfix, in my case postfix is not running.

Kindly Help

Thanks & Regards
Sachin

**Comment by willpower232** [comment]
is there a firewall on your server or from your server provider that needs opening?

**Reply by ttechnomarkets** [reply]
Hi..
Thank you for the reply.. I am not sure about it, I am using OVH dedicated server. I have contacted them too. 
Waiting for there reply as well.
Once i get same..  I will update. 

Thank you

**Reply by ttechnomarkets** [reply]
Hi There is an update...
I have contacted VPS and got reply from them saying they do not block any SMTP ports.
and I did login to one of my cloud windows VPS and I checked, the telnet is working.
Also i checked with https://www.smtper.net/, the email went smooth. its working.

But only when I do telnet or ping from my local system using multiple ISP's still i am unable to connect SMTP server postal using outlook or telnet.

**Reply by willpower232** [reply]
it sounds like your home ISPs (or even your operating systems) are blocking SMTP ports then, Postal is not intended for personal use, rather to enable software hosted on servers to send email so it depends what you were hoping to do.

**Reply by iamjohnseun** [reply]
I have the same issue and i opened #1697 i can confirm that i don't have a firewall and OS is Ubuntu 20.04 on Contabo VPS with port 25 open.
